### PR TITLE
Docs: added topKWeighted description

### PR DESCRIPTION
**Pull request by godfreyd**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

### CODE REVIEW

File: docs/ru/query_language/agg_functions/reference.md [hunk 1]
```diff
@@ -1010,6 +1010,44 @@ FROM ontime
 └─────────────────────┘
 ```
 
+## topKWeighted {#topkweighted}
+
+Возвращает массив наиболее часто встречающихся значений в указанном столбце. Результирующий массив упорядочен по убыванию веса значения. Если передан опциональный параметр `N`, то будет создан срез с n элементами.
```
Comment (alexey-milovidov) [inline docs/ru/query_language/agg_functions/reference.md]: Looks plain wrong.

Comment (godfreyd) [reply docs/ru/query_language/agg_functions/reference.md]: How true?

File: docs/ru/query_language/agg_functions/reference.md [hunk 2]
```diff
@@ -1010,6 +1010,44 @@ FROM ontime
...
+
+```sql
+topKWeighted(N)(x, weight)
+```
+
+**Параметры**
+
+- `N` — Количество возвращаемых значений. Опциональный параметр. [UInt8](../../data_types/int_uint.md).
+- `x` — Столбец.
+- `weight` — Вес. [UInt8](../../data_types/int_uint.md).
```
Comment (alexey-milovidov) [inline docs/ru/query_language/agg_functions/reference.md]: Also wrong.

File: docs/ru/query_language/agg_functions/reference.md (hunk 1 above)
Comment (alexey-milovidov) [reply docs/ru/query_language/agg_functions/reference.md]: Not true at all.

File: docs/en/query_language/agg_functions/reference.md [hunk 3]
```diff
@@ -996,6 +996,44 @@ FROM ontime
 └─────────────────────┘
 ```
 
+## topKWeighted {#topkweighted}
+
+Returns an array of the most frequent values in the specified column. The resulting array is sorted by weight of value. If the optional parameter `N` is passed, сreates a slice of array with n elements.
```
Comment (alexey-milovidov) [inline docs/en/query_language/agg_functions/reference.md]: Wrong.

Slice is always created. If parameter N is unspecified, default value 10 is used.

File: docs/en/query_language/agg_functions/reference.md [hunk 4]
```diff
@@ -996,6 +996,44 @@ FROM ontime
...
+**Syntax**
+
+```sql
+topKWeighted(N)(x, weight)
+```
+
+**Parameters**
+
+- `N` — The number of elements to return. Optional parameter. [UInt8](../../data_types/int_uint.md).
+- `x` – The column.
```
Comment (alexey-milovidov) [inline docs/en/query_language/agg_functions/reference.md]: Wrong.

**Pull request merged**